### PR TITLE
fix(google): Made instrumentation a little more consistent

### PR DESCRIPTION
**Pull request by ewiseblatt**
@jtk54

### CODE REVIEW

**Comment by ewiseblatt** [comment]
Can you look at the last delta I did?
I wasnt able to submit this earlier because Travis never completed for some reason.
While trying to retrigger it I surveyed all the call sites and found another cluster of changes.

File: clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/ops/loadbalancer/DeleteGoogleSslLoadBalancerAtomicOperation.groovy [hunk 1]
```diff
@@ -173,7 +173,7 @@ class DeleteGoogleSslLoadBalancerAtomicOperation extends DeleteGoogleLoadBalance
         "Health check $healthCheckName",
         project,
         task,
-        BASE_PHASE,
+        [action: 'delete', operation: 'compute.healthChecks.delete', phase: BASE_PHASE, (TAG_SCOPE): SCOPE_GLOBAL],
```
Comment (jtk54) [inline clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/ops/loadbalancer/DeleteGoogleSslLoadBalancerAtomicOperation.groovy]: If these things are all of the same form, using an object here could be helpful.

Comment (ewiseblatt) [inline clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/ops/loadbalancer/DeleteGoogleSslLoadBalancerAtomicOperation.groovy]: I was tempted to do a big refactoring here but I dont want to break it. Instead I saved the fight for another day when I have more time. There are wrappers upon wrappers here and redundant parameters to go around. I have some thoughts about how to approach this
but probably not the time for a few weeks.

Comment (jtk54) [inline clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/ops/loadbalancer/DeleteGoogleSslLoadBalancerAtomicOperation.groovy]: SGTM.

**Pull request merged**